src/App.js

```diff
@@ -1,31 +1,37 @@
-import Calculator from './components/calculator';
-import GetQuote from './components/getQuote';
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Link, Routes,
+} from 'react-router-dom';
+import CalculatorComponent from './routes/CalculatorComponent';
+import Quote from './routes/GetQuoteComponent';
+import Home from './routes/HomeComponent';
 import './App.css';
 
 function App() {
   return (
-    <div>
-      <nav className="navbar">
-        <div className="appName">
-          <h1>Math Magicians</h1>
-        </div>
-        <ul className="nav-list">
-          <li><a href="#home">Home</a></li>
-          <li>|</li>
-          <li><a href="#calculator">Calculator</a></li>
-          <li>|</li>
-          <li><a href="#quote">Quote</a></li>
-        </ul>
-      </nav>
-      <div className="contain">
-        <div className="quoteApp">
-          <GetQuote />
-        </div>
-        <div className="calculatorApp">
-          <Calculator />
+    <Router>
+      <div>
+        <nav className="navbar">
+          <div className="appName">
+            <h1>Math Magicians</h1>
+          </div>
+          <ul className="nav-list">
+            <li><Link to="/">Home</Link></li>
+            <li>|</li>
+            <li><Link to="/calculator">Calculator</Link></li>
+            <li>|</li>
+            <li><Link to="/quote">Quote</Link></li>
+          </ul>
+        </nav>
```

Comment:
- [ OPTIONAL BUT HIGHLY RECOMMENDED ] Hello! 👋 You've done an excellent job with your React Router DOM routing implementation. I'd like to highlight a principle that's essential for clean and reusable code – the Single Responsibility Principle. In your current code, it appears that this principle is being inadvertently violated.

- You've placed the implementation of the navbar inside the 'App' component. In the ideal application structure, the 'App' component should primarily serve as the top-level component for rendering your application. It's typically not the responsibility of 'App' to create the application's navbar.
- For better code organization and reusability, it's recommended to create a separate 'Navbar' component. This way, you can import and use the 'Navbar' component wherever it's needed throughout your application. This separation of concerns makes your code more modular, easier to maintain, and adheres to the principle of single responsibility.

In your upcoming projects and milestones, I would suggest paying close attention to this aspect of code organization to maintain a more structured and maintainable codebase. 🚀👨‍💻